tests/testsuite/resolve.rs

```diff
@@ -17,12 +17,7 @@ use proptest::prelude::*;
 
 proptest! {
     #![proptest_config(ProptestConfig {
-        cases:
-            if env::var("CI").is_ok() {
-                256 // we have a lot of builds in CI so one or another of them will find problems
-            } else {
-                1024 // but locally try and find it in the one build
-            },
+        cases: 256,
```

Comment:
Why did you choose to change this?

I assume it was running to slowly locally. If that is correct can we atleast add back in a comment on how running repeatedly or increasing this number will increase the chance of finding a bug at the expense of time?

Reply:
Ah yeah I found this was taking ~4m or so on my local machine which made a full test run sort of unbearable unfortunately :(. And sure I'll add a comment!

Reply:
That makes sense. It is new and we are still experimenting with how it fits into the workflow. I am ok with running fewer iterations, for the better ergonomics of running the test.